### PR TITLE
Adding donutor after fixes

### PR DESCRIPTION
**Pull request by wolnikowa**
Nie zmieniłam tylko tej generycznej tabeli bo  to było w sumie zrobione przez Ale i nie chce teraz mieszać

### CODE REVIEW

File: src/components/DishonorableTable/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './style.scss'
+import HeaderD from './components/Header/index.js';
+import BodyD from './components/Body/index.js';
+  
+
+  const TableD = () => {
```
Comment (rlayeh) [inline src/components/DishonorableTable/index.js]: remove this table and use generic table.

File: src/components/DishonorableTable/components/Body/index.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        addedby:'Ryszard Jakielski',
+        dateadd:'20-03-2021',
+        datedonut: '21-03-2021',
+        lateness: '36'
+    }
+  ]
+
+
+
+const BodyD = () => {
```
Comment (rlayeh) [inline src/components/DishonorableTable/components/Body/index.js]: This component can be removed.

File: src/components/DishonorableTable/components/Header/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const HeaderD = () => {
```
Comment (rlayeh) [inline src/components/DishonorableTable/components/Header/index.js]: this component can be removed.

File: src/components/DishonorableTable/index.js (hunk 1 above)
Comment (rlayeh) [reply src/components/DishonorableTable/index.js]: `<Table data={data} columns={columnsDef} />`
use it on this page.

**Review by rlayeh — CHANGES_REQUESTED** [review]

File: src/components/DishonorableTable/index.js (hunk 1 above)
Comment (wolnikowa) [reply src/components/DishonorableTable/index.js]: Alicja already did that while me and Julia were taking a break. I simply deleted this whole 'dishonorable' folder, bc the generic table for that is used in components/Pages/Dishonorable/index.

**Review by rlayeh — APPROVED** [review]

**Pull request merged**